src/org/jetbrains/plugins/scala/codeInspection/convertNullInitializerToUnderscore/ConvertNullInitializerToUnderscore.scala

```diff
@@ -0,0 +1,59 @@
+package org.jetbrains.plugins.scala.codeInspection.convertNullInitializerToUnderscore
+
+import com.intellij.codeInspection.{ProblemHighlightType, ProblemsHolder}
+import com.intellij.openapi.project.Project
+import org.jetbrains.plugins.scala.codeInspection.convertNullInitializerToUnderscore.ConvertNullInitializerToUnderscore._
+import org.jetbrains.plugins.scala.codeInspection.{AbstractFixOnPsiElement, AbstractInspection, InspectionBundle}
+import org.jetbrains.plugins.scala.lang.lexer.ScalaTokenTypes
+import org.jetbrains.plugins.scala.lang.psi.api.expr.ScExpression
+import org.jetbrains.plugins.scala.lang.psi.api.statements.ScVariableDefinition
+import org.jetbrains.plugins.scala.lang.psi.impl.ScalaPsiElementFactory
+import org.jetbrains.plugins.scala.lang.psi.types.api
+
+/**
+  * Created by a.tsukanov on 26.05.2016.
+  */
+class ConvertNullInitializerToUnderscore extends AbstractInspection(inspectionId, inspectionName) {
+  private def registerProblem(holder: ProblemsHolder, variable: ScVariableDefinition): Unit = {
+    val fix = new ConvertNullInitializerToUnderscoreQuickFix(variable)
```

Comment:
Constant `ProblemHighlightType.WARNING` does not exists, so, I removed `highlightType` parameter as in AutoTuplingInspection